packages/kolibri-common/components/SyncSchedule/EditDeviceSyncSchedule.vue

```diff
@@ -0,0 +1,395 @@
+<template>
+
+  <ImmersivePage
+    :appBarTitle="$tr('toolbarHeader')"
+    :route="backRoute"
+    :icon="icon"
+  >
+    <KPageContainer v-if="device" :style="pageHeight">
+      <KGrid gutter="48" class="edit-sync-schedule">
+
+        <KGridItem>
+          <h1>{{ $tr('editSyncScheduleSubTitle') }}</h1>
+        </KGridItem>
+
+        <KGridItem>
+          <p>{{ device.device_name }} </p>
+        </KGridItem>
+
+        <KGridItem class="select-element">
+          <div
```

Comment:
To get the desired responsive behaviour here, you could use a nested `KGrid` within the parent `KGrid` - this should also enforce standard spacing between the `KSelect`s if they are each inside their own `KGridItem`.